rest-api-spec/src/yamlRestTest/resources/rest-api-spec/test/search.aggregation/49_range_timezone_bug.yml

```diff
@@ -0,0 +1,55 @@
+setup:
+    - do:
+        indices.create:
+            index: test
+            body:
+                settings:
+                    number_of_replicas: 0
+                mappings:
+                    properties:
+                        mydate:
+                            type: date
+                            format: "uuuu-MM-dd'T'HH:mm:ss.SSSSSSSSSZZZZZ"
+
+    - do:
+        cluster.health:
+            wait_for_status: green
+
+    - do:
+        index:
+            index: test
+            id: 1
+            body: { "mydate": "2021-08-12T01:00:00.000000000+02:00" }
+
+    - do:
+        indices.refresh: {}
+
+---
+"time zone bug":
+    - do:
+        search:
+            rest_total_hits_as_int: true
+            body: {
+                      "query": {
+                        "match_all": {}
+                      },
+                      "aggregations": {
+                        "myagg": {
+                          "date_range": {
+                            "field": "mydate",
+                            "ranges": [
+                              {
+                                "from": "2021-08-12T00:00:00.000000000+02:00",
+                                "to": "2021-08-12T02:00:00.000000000+02:00"
+                              }
+                            ]
+                          }
+                        }
+                      }
+                    }
+    - match: { hits.total: 1 }
+    - length: { aggregations.myagg.buckets: 1 }
+    - match: { aggregations.myagg.buckets.0.from: 1628719200000 }
```

Comment:
I wonder if we could add human readable date in a comment to help with readability?